### PR TITLE
bugfixes for main(container)

### PR DESCRIPTION
**Pull request by Kachyr**
#149 task
bugfix for padding-top of main container
also some bugfixes for details page

### CODE REVIEW

File: src/App.scss [hunk 1]
```diff
@@ -40,10 +40,10 @@ body {
 .main {
   flex: 1 1 auto;
   overflow: hidden;
-  padding: 140px;
```
Comment (AnnaBolshak) [inline src/App.scss:43]: You are removing bottom padding too with this changes, and the footer overlaps the main content

Comment (Kachyr) [reply src/App.scss:43]: fixed it by adding padding-bottom

**Review by AnnaBolshak — APPROVED** [review]
I think also is needed review from your colleagues for all PRs

**Comment by AnnaBolshak** [comment]
Please check the comment
The review rom your colleagues for all PRs  is needed

**Review by oleksandrstarov — APPROVED** [review]

**Review by Dmytro1991ua — APPROVED** [review]

File: src/components/MovieDetails.tsx [hunk 2]
```diff
@@ -68,17 +68,24 @@ const MovieDetails = ({ match }: MovieDetailsProps) => {
...
+  }, []);
 
-  const renderMovieInfo = (header: string, info: string | number): ReactNode => {
+  const renderMovieInfo = (
+    header: string,
+    info: string | number
+  ): ReactNode => {
     return (
       <tr>
-        <td><h4>{header}: </h4></td>
```
[the remark this answers is not in the document]
Comment (Kachyr) [reply src/components/MovieDetails.tsx:77]: In #91 PR I have already made a component "detail" for this case now we just need to replace it in task #160

**Pull request merged**